src/modules/endpoint/endpoints/feedback/categoriesFeedback.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+
+import { apiUrl } from '../../constants'
+import ParamMissingError from '../../errors/ParamMissingError'
+import FeedbackEndpointBuilder from '../../FeedbackEndpointBuilder'
+
+const CATEGORIES_FEEDBACK_ENDPOINT_NAME = 'categoriesFeedback'
+const POSITIVE_RATING_STRING = 'up'
+const NEGATIVE_RATING_STRING = 'down'
+
+export default new FeedbackEndpointBuilder(CATEGORIES_FEEDBACK_ENDPOINT_NAME)
+  .withParamsToUrlMapper((params): string => {
+    if (!params.city) {
+      throw new ParamMissingError(CATEGORIES_FEEDBACK_ENDPOINT_NAME, 'city')
+    }
+    if (!params.language) {
+      throw new ParamMissingError(CATEGORIES_FEEDBACK_ENDPOINT_NAME, 'language')
+    }
+    return `${apiUrl}/${params.city}/${params.language}/wp-json/extensions/v3/feedback`
+  })
+  .withParamsToBodyMapper((params): FormData => {
+    if (params.id === undefined) {
+      throw new ParamMissingError(CATEGORIES_FEEDBACK_ENDPOINT_NAME, 'page id')
+    }
+
+    if (params.isPositiveRating === undefined && !params.comment) {
```

Comment:
Using undefined in a type is a bit unsecure. `isPositiveRating` should be a boolean so it only can be true or false.
In this code `isPositiveRating` has the type `boolean|void`

Same goes for id etc.